src/server/rest/v1/service/UserService.ts

```diff
@@ -1280,7 +1281,7 @@ export default class UserService {
         firstName: user.First_Name,
         email: user.Email,
         importedBy: req.user.id,
-        status: UserImportStatus.UNKNOWN
+        status: ImportStatus.UNKNOWN
       };
       await UserStorage.saveImportedUser(req.user.tenantID, newUploadedUser);
```

Comment:
We should check the data with ajv (performant) before we save it in the DB

Reply:
Done